### PR TITLE
Document coords attribute of WCSAxes

### PR DESCRIPTION
**Pull request by dstansby**
Does what it says in the title - since the `.coords` attribute is used in at least one example (https://docs.astropy.org/en/stable/visualization/wcsaxes/custom_frames.html#frame-properties), it seems worth documenting here.

### CODE REVIEW

**Review by larrybradley — APPROVED** [review]
Thanks.

**Comment by larrybradley** [comment]
Needs to be rebased before merging.

**Review by astrofrog — APPROVED** [review]

**Comment by pllim** [comment]
The mpl job named changed, hence it was out of sync with current branch protection rules and grayed out the merge button. I overrode it and merged since this is a doc change only. Thanks for your patience!

**Pull request merged**
